charts/dapr/crds/configuration.yaml

```diff
@@ -152,6 +152,12 @@ spec:
                 properties:
                   samplingRate:
                     type: string
+                  zipkin:
+                    type: object
+                    description: ZipkinSpec defines the Zipkin trace configurations
```

Comment:
Would add a space here   "description: Zipkin spec defines the Zipkin trace configurations"
Or should this be like this? "description: Defines the Zipkin trace configuration settings"

Reply:
I like `Defines the Zipkin trace configuration settings`. Updating.

Reply:
Done now